src/Microsoft.AspNet.Authentication.JwtBearer/JwtBearerHandler.cs

```diff
@@ -37,7 +37,7 @@ internal class JwtBearerHandler : AuthenticationHandler<JwtBearerOptions>
 
                 if (receivingTokenContext.Skipped)
                 {
-                    return AuthenticateResult.Success(ticket: null);
+                    return AuthenticateResult.Failed("Skipped.");
```

Comment:
Skipped != Failed. Skipped is supposed to mean "This wasn't for me, try the next middleware". You may need to promote Skipped into AuthenticateResult to make that work.